### PR TITLE
Enable/disable Zappr checks

### LINKED ISSUE #13: Enable status checks in zappr
Once zalando/zappr#343 is merged one be able to use Oakkeeper to automatically enable Zappr checks in repositories.

Needed:
* Option for Zappr URL, defaults to `https://zappr.opensource.zalan.do`
* List of checks that should be enabled

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #13

### CODE REVIEW

**Comment by prayerslayer** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -38,6 +40,11 @@ You can also directly upload a local `.zappr.yaml` or an issue/PR template to ev
 * Path to `PULL_REQUEST_TEMPLATE.md`: `--pr-template-path` or `OK_PULL_REQUEST_TEMPLATE_PATH`
 * Upload type: `--upload-type` or `OK_UPLOAD_TYPE`. Might be `commit` (commit to default branch) or `pr` (open pull request).
 
+To enable/disable checks in Zappr use these:
+
+* `--enable`: Enables a Zappr check (currently `approval`, `commitmessage` or `autobranch`) for a repository
```
Comment (gagoman) [inline README.md]: May be we need to receive a list of checks to be enabled?

**Comment by gagoman** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**